### ISSUE TITLE
Problems about the “Computes the accuracy ”

### ISSUE THREAD

**Issue by tangtaogo** [CupidJay/MoCov3-pytorch#1]
Thanks your fast implementation of mocov3.
But I confused about the "acc1/acc5 are (K+1)-way contrast classifier accuracy" in your code. Can I get correct acc through `output, target = model(im_q=images[1], im_k=images[0])`? 
In other words, how can I get the kNN monitor in the paper？

**Comment by CupidJay** [comment]
Thanks for your interest. The classifier accuracy was the metric used in MoCo and MoCov2, which means the instance discrimination accuracy in the memory bank (please refer to the official paper and code). MoCov3 was trained in a different way and I commented these lines in my code and only recorded the loss during training for MoCov3. 
For KNN monitor, I have no plan to update and you can implement this function yourself.

**Comment by tangtaogo** [comment]
Thank you for your kind reply,  I will try to implement this function by myself.
